compiler/natives/src/reflect/reflect.go

```diff
@@ -694,6 +694,39 @@ func Copy(dst, src Value) int {
 	return js.Global.Call("$copySlice", dstVal, srcVal).Int()
 }
 
+func methodReceiver(op string, v Value, i int) (_ *rtype, t *funcType, fn unsafe.Pointer) {
```

Comment:
I'm not sure this PR's intention. Does this PR remove unnecessary things? I was wondering why you added new code.

Reply:
There were two copies of these functions, one for Go 1.11.0 in reflect_go111.go and another for Go 1.11.1+ in reflect_go1111.go.

The copy in reflect_go111.go is no longer ever needed, because we don't support Go 1.11.0 in GopherJS 1.12. This PR removes it. It moves the actively used code from reflect_go1111.go back into reflect.go, since there's no need for it to be in a separate file anymore.

It effectively reverts PR #865, which added complexity that was needed at the time to fix issue #865.